### PR TITLE
NickeManarin.ScreenToGif version 2.33

### PR DESCRIPTION
**Pull request by NickeManarin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21007)

### CODE REVIEW

File: manifests/n/NickeManarin/ScreenToGif/2.33/NickeManarin.ScreenToGif.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.33
```
Comment (OfficialEsco) [inline manifests/n/NickeManarin/ScreenToGif/2.33/NickeManarin.ScreenToGif.installer.yaml:5]:
```suggestion
PackageVersion: 2.33.0
```

File: manifests/n/NickeManarin/ScreenToGif/2.33/NickeManarin.ScreenToGif.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.33
```
Comment (OfficialEsco) [inline manifests/n/NickeManarin/ScreenToGif/2.33/NickeManarin.ScreenToGif.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.33.0
```

File: manifests/n/NickeManarin/ScreenToGif/2.33/NickeManarin.ScreenToGif.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.33
```
Comment (OfficialEsco) [inline manifests/n/NickeManarin/ScreenToGif/2.33/NickeManarin.ScreenToGif.yaml:5]:
```suggestion
PackageVersion: 2.33.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You have to use the same version which appears in the Control panel
image
Remember to change the version folder name.
Also, i see that you can install it for all users and current user only, what are the switches for those?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/21007#pullrequestreview-707063824

**Comment by NickeManarin** [comment]
@OfficialEsco Thanks. Busy week, could only work on today (but someone else already fixed it).

**Comment by OfficialEsco** [comment]
> @OfficialEsco Thanks. Busy week, could only work on today (but someone else already fixed it).

Yeeeah people don't tend to respond... So if it takes too long someone active will fix it, at least when its a simple fix like this 🙄

**Pull request closed without merging**